### ISSUE TITLE
BUG : Improper check of old pip version

### ISSUE THREAD

**Issue by lxj616** [andresriancho/w3af#13329]
On Ubuntu 15.10 most recently , can not run w3af as follows

I used the most recent pip from Ubuntu repository as well as sudo pip install --upgrade pip

but It still won't work

```console
alkaid@alkaid-VirtualBox:~/w3af$ pip -V
pip 1.5.6 from /usr/lib/python2.7/dist-packages (python 2.7)
alkaid@alkaid-VirtualBox:~/w3af$ ./w3af_gui 
An old pip version was detected. We recommend a pip update before continuing:
    sudo pip install --upgrade pip
alkaid@alkaid-VirtualBox:~/w3af$ python
Python 2.7.10 (default, Oct 14 2015, 16:09:02) 
[GCC 5.2.1 20151010] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from pip._vendor.packaging.version import Version
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named _vendor.packaging.version
>>> 
```

**Comment by andresriancho** [comment]
Looks like my pip requirement test is broken. Since you've identified where the bug is, would you consider sending a PR?

**Comment by lxj616** [comment]
I took a deeper look into this and find something odd , This issue is related to pip itself : https://github.com/pypa/pip/issues/2964

And pip official says this is a debian patch causing trouble

Maybe I should try reporting this to debian community ...

**Comment by lxj616** [comment]
Fixed in ubuntu 16.04 LTS , It is indeed a debian pip patch issue , since 16.04 updated pip version by default (Still can not upgrade, but we don't need to upgrade anymore)

**Comment by andresriancho** [comment]
Thanks for all the details!
